src/main/java/parser/Parser.java

```diff
@@ -542,5 +549,25 @@ private static void eventTentativeCommand(String todoTask1) throws DukeException
             throw DukeException.EMPTY_TASK_IN_EVENT_TENTATIVE;
         }
     }
+
+    /**
+     * Find all the deadlines/events on the user specified date
+     * Then print them out for user
+     * Command: schedule <date>
+     */
+    private static void scheduleCommand(String s) {
```

Comment:
This function will get all the deadlines/events on the date user enter. Seems to work.